src/clusterfuzz/_internal/bot/fuzzers/centipede/engine.py

```diff
@@ -267,8 +267,13 @@ def reproduce(self, target_path, input_path, arguments, max_time):  # pylint: di
           f':{constants.RSS_LIMIT_MB_FLAGNAME}={rss_limit}'
           f':{constants.TIMEOUT_PER_INPUT_FLAGNAME}={timeout}:')
 
-    runner = new_process.UnicodeProcessRunner(sanitized_target, [input_path])
-    result = runner.run_and_wait(timeout=max_time)
+    if sanitized_target.endswith(('_fuzztest', '_fuzz_test')):
```

Comment:
Can we do something else less hacky here? Can you check for FUZZTEST_MODE env var? I can set the job to use it.

Reply:
I can do that and it would be fine for V8. For Chromium it would require some more follow up work, since there's currently one shared job for fuzztest and legacy-libfuzzer centepede. Setting a flag on the job level would require this to have two jobs and probably some more logic to not pick fuzztests in one job and only fuzztests in the other.

Reply:
But maybe Chromium is fine to switch to the googlefuzztest engine anyways? Then no further fixes are required.

Reply:
Done. PTAL